provider/oauth2/tests.py

```diff
@@ -126,6 +126,20 @@ def test_authorization_requires_supported_response_type(self):
         response = self.client.get(self.auth_url2())
         self.assertEqual(200, response.status_code)
 
+    def test_token_authorization_redirects_to_correct_uri(self):
+        self.login()
+
+        response1 = self.client.get(self.auth_url() + '?client_id=%s&response_type=token' % self.get_client().client_id, follow=True)
```

Comment:
Is `.format` preferred over keeping consistent style with the repository itself?

The reason for not using data was because other API calls (see method just above this one) didn't.